docs/area-owners.md

```diff
@@ -93,6 +93,7 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 | area-System.Dynamic.Runtime                    | @jaredpar            | @cston @333fred                                     | Archived component - limited churn/contributions (see [#27790](https://github.com/dotnet/runtime/issues/27790))                                                                                                                                                                           |
 | area-System.Formats.Asn1                       | @jeffhandley         | @dotnet/area-system-formats-asn1                    | Consultants: @bartonjs @vcsjones                                                                                                                                                                                                                                                          |
 | area-System.Formats.Cbor                       | @jeffhandley         | @dotnet/area-system-formats-cbor                    | Consultants: @bartonjs @vcsjones                                                                                                                                                                                                                                                          |
+| area-System.Formats.Nrbf                       | @jeffhandley         | @dotnet/area-system-formats-nrbf                    | Consultants: @bartonjs @grabyourpitchforks                                                                                                                                                                                                                                                |
```

Comment:
Nit: looks like @bartonjs is a member of the team so he doesn't need to be listed as consultant.

Reply:
Yeah, but he's probably going to try to get out of the team after not too long; but is still curious enough to lurk... so he doesn't have a problem with it.

Hmm, he's talking about himself in the 3rd person again...